module-1/lab-api-scavenger-game/your-code/challenge-1.ipynb

```diff
@@ -0,0 +1,224 @@
+{
```

Comment:
Good job!!
Before creating a dataframe you should always normalize JSONs. JSONs object obtained in requests usually throw nested JSONs (JSON within JSON). When creating a pandas dataframe without normalizing you can lose important information that remains as a dictionary in each record of your dataframe.